supporter.py

```diff
@@ -73,7 +73,7 @@ def create_post(
     ) -> tuple[bool, dict]:
         # Create post ID and get timestamp
         post_id = str(uuid.uuid4())
-        ts = timestamp(1).copy()
+        ts = time()

```

Comment:
A small nit; `time.time()` looks better here

Reply:
i prefer importing as little as possible whenever possible (even though that doesn't make a difference in python due to its design) and i also prefer explicitly stating what i'm importing from a module whenever possible (no `from x import *` or `import x`